spec/requests/api/v1/merchants/business_intel_spec.rb

```diff
@@ -1,24 +1,34 @@
 require 'rails_helper'
 
 describe 'Merchants API' do
-  it 'returns the total revenue for that merchant across successful transactions' do
-    merchant = create(:merchant)
-    item_1 = create(:item, merchant_id: merchant.id)
-    item_2 = create(:item, merchant_id: merchant.id)
+  before(:each) do
+    @merchant = create(:merchant)
+    item_1 = create(:item, merchant_id: @merchant.id)
+    item_2 = create(:item, merchant_id: @merchant.id)
     invoice = create(:invoice)
-    invoice_item_1 = create(:invoice_item, item: item_1, invoice: invoice)
+    @invoice_item_1 = create(:invoice_item, item: item_1, invoice: invoice)
     invoice_item_2 = create(:invoice_item, item: item_1, invoice: invoice)
```

Comment:
I think for invoice items 3, 4, and 5, you can say something like the following:
`other_invoice_items = create_list(:invoice_item, 3, item: item_2, invoice: invoice)`
It should create three invoice items that belong to item_2 and invoice.